### PR TITLE
Fix README.md and locales

### PR DESCRIPTION
**Pull request by athdesk**
Adds missing languages' support in README.md and fixes a typo in the italian locale

### CODE REVIEW

**Comment by athdesk** [comment]
closing due to pull request being incomplete, didn't test the locale

**Comment by athdesk** [comment]
everything works now apparently, might have been a problem with my xcode instance
i'm not sure if this works for everyone, so please test this before pulling

File: MuteSpotifyAds/Assets.xcassets/AppIcon.appiconset/Contents.json [hunk 1]
```diff
@@ -1,63 +1,63 @@
 {
-  "images": [
+  "images" : [
     {
-      "size": "16x16",
-      "idiom": "mac",
-      "filename": "icon_16.png",
-      "scale": "1x"
+      "size" : "16x16",
```
Comment (qmnl) [inline MuteSpotifyAds/Assets.xcassets/AppIcon.appiconset/Contents.json]: Please do not change the formatting of JSON files.

File: MuteSpotifyAds/Assets.xcassets/AppIcon.appiconset/Contents.json [hunk 2]
```diff
@@ -1,63 +1,63 @@
...
-      "scale": "2x"
+      "idiom" : "mac",
+      "size" : "512x512",
+      "scale" : "2x"
     }
   ],
-  "info": {
-    "version": 1,
-    "author": "xcode"
+  "info" : {
```
Comment (qmnl) [inline MuteSpotifyAds/Assets.xcassets/AppIcon.appiconset/Contents.json]: Furthermore, please do not update the binary images.

File: MuteSpotifyAds/it.lproj/MainMenu.strings [hunk 3]
```diff
@@ -9,22 +9,25 @@
 "DTq-sR-yDf.title" = "🔖 Licenza";
 
 /* Class = "NSMenuItem"; title = "≡  Song log"; ObjectID = "GPc-bM-QPt"; */
-"GPc-bM-QPt.title" = "≡  Log delle cazoni";
-
-/* Class = "NSMenuItem"; title = "© 2019 Simon Meusel"; ObjectID = "IUf-ND-5vk"; */
```
Comment (qmnl) [inline MuteSpotifyAds/it.lproj/MainMenu.strings:14]: Why did you remove this part of the file?

**Review by qmnl — CHANGES_REQUESTED** [review]
Thank you for fixing the README.md and the Itatlian lanuage.

But you also changed the formatting of one JSON file and changed the `.png` files. Please revert those changes.

Alternatively, I can pick out the changes and put them in a separate commit, if you want.

This PR contains a lot of different changes. In general it is recommended to only change specific parts in one PR.

File: MuteSpotifyAds/Assets.xcassets/AppIcon.appiconset/Contents.json (hunk 1 above)
Comment (athdesk) [reply MuteSpotifyAds/Assets.xcassets/AppIcon.appiconset/Contents.json]: i believe xcode automatically did this when i changed my icons to match the ones on github, i'll do a commit to revert those

File: MuteSpotifyAds/it.lproj/MainMenu.strings (hunk 3 above)
Comment (athdesk) [reply MuteSpotifyAds/it.lproj/MainMenu.strings:14]: git marks that as removed, it really is just in a different place, i believe because of the way the xcode localisation tool saved it

File: MuteSpotifyAds/Assets.xcassets/AppIcon.appiconset/Contents.json (hunk 2 above)
Comment (athdesk) [reply MuteSpotifyAds/Assets.xcassets/AppIcon.appiconset/Contents.json]: Also did this because using git clone somehow managed to pull 0-bytes long icon files, and i did manually fix them by downloading them with wget so it could build, so these can be left out too

**Comment by athdesk** [comment]
I reverted everything in the `AppIcon.appiconset` package, which were the changes to `Contents.json` and the image files
I should really have commited just README and the localisation files, instead i did `git add .` thinking it wouldn't affect the icon-related files, as all i did with them was replacing the corrupted ones that were cloned with copies from the github web service, ultimately i'm sorry for the inconvenience, and now the pull request is much cleaner

File: MuteSpotifyAds/it.lproj/MainMenu.strings (hunk 3 above)
Comment (athdesk) [reply MuteSpotifyAds/it.lproj/MainMenu.strings:14]: now i'm noticing that it may have done this because the ObjectID for `Class = "NSMenuItem"; title = " ©  Simon Meusel";  ` was wrong, in fact before (IUf-ND-5vk) it didn't match the one in MainMenu.xib, now (vrP-PM-sCp) it does

**Review by qmnl — APPROVED** [review]

**Comment by qmnl** [comment]
Thank you. This is going to be in the next release.
